patroni/ctl.py

```diff
@@ -430,31 +446,85 @@ def ctl_load_config(cluster_name, config_file, dcs):
     return config, dcs, cluster
 
 
+def check_response(response, member_name, action_name, silent_success=False):
+    if response.status_code != 200:
```

Comment:
API may return you 202 for the scheduled action.
https://github.com/zalando/patroni/blob/master/patroni/api.py

In general, I'd avoid complaining on anything less than 400.